### DISCUSSION TITLE
Auto Merge Error

### DISCUSSION THREAD

**Discussion by MrBotDeveloper** [renovatebot/renovate#10263] [Request Help] [answered]
Auto merge not working after setting config it is opening an issue [this one](https://github.com/MrBotDeveloper/telegram-pdf-bot/issues/57) 

And my renovate.json looks like 
```
{
  "extends": ["config:base"],
  "labels": ["dependencies"],
  "stabilityDays": 3,
  "prCreation": "not-pending",
  "schedule": ["after 6pm every day", "every weekend"],
  "timezone": "Australia/Sydney",
  "packageRules": [
    {
      "matchPaths": [".github/workflows/*.yml"],
      "labels": ["github-actions"],
      "automerge": true
    },
    { "matchFiles": ["Dockerfile"], "labels": ["docker"], },
  ]
}
```

**Comment by rarkins** [comment, marked as answer]
You have a JSON syntax error, unrelated to automerge. Remove this comma:

![image](https://user-images.githubusercontent.com/6311784/120347000-2f893380-c2fc-11eb-8759-e4bfe536bebc.png)

Then put your config into jsonlint.com to check it validates
